Primo.js

```diff
@@ -456,7 +475,10 @@ function stripAuthor(str) {
 		// Remove year
 		.replace(/\s*,?\s*\(?\d{4}-?(\d{4})?\)?/g, '')
 		// Remove things like (illustrator). TODO: use this to assign creator type?
-		.replace(/\s*,?\s*[\[\(][^()]*[\]\)]$/, '');
+		.replace(/\s*,?\s*[\[\(][^()]*[\]\)]$/, '')
+		// The full "continuous" name uses no separators, which need be removed
+		// cf. "Luc, Jean André : de (1727-1817)"
+		.replace(/ :/, "");
```

Comment:
I would suggest a slightly different replacement here for dealing with different possible spacing options:
```js
.replace(/\s+:\s+/, " ");
```